docs/diagrams/ListCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,85 @@
+@startuml ListCommandSequenceDiagram
+'https://plantuml.com/sequence-diagram
+
+!include style.puml
+
+box "Command\n" COMMAND_COLOR_T1
+Participant ":ListCommand" as list COMMAND_COLOR
+end box
+
+box "Food\n" FOOD_COLOR_T1
+Participant ":Fridge" as fridge FOOD_COLOR
+Participant ":Food" as food FOOD_COLOR
+end box
+
+-> list: execute()
+activate list
+
+alt list all
+        list -> list: getListAllMessage()
+        activate list
+        loop food in fridge
+            list-> fridge: getFoodDescription()
+            activate fridge
+
+            fridge -> food: getFood()
+                activate food
+                return
+                return
+                deactivate food
+
+            deactivate fridge
+        end
+
+<--list: list of all Food
```

Comment:
I think you can just return without a text since the listing is actually a void method where all their getters are actually passed to the UI.

Reply:
This is true, but would it be clearer for the reader to leave this expected output here, or to leave it blank for accuracy?

Reply:
After some thought, I think it would be clearer to leave this as it is, because I need at least one path that tells us that the list is being output, which was not indicated anywhere else

Reply:
I have thought of a solution that would solve both issues in my latest commit.